src/clusterfuzz/_internal/base/utils.py

```diff
@@ -1031,7 +1031,9 @@ def is_oss_fuzz():
 
 def is_chromium():
   """If this is an instance of chromium fuzzing."""
-  return default_project_name() in ('chromium', 'chromium-testing')
+  return default_project_name() in ('chromium', 'chromium-testing',
```

Comment:
Maybe we can make an env var or config that allows a project to specify it is chromium instead of hardcoding who is?